### ISSUE TITLE
sample csv?

### ISSUE THREAD

**Issue by jordibruin** [guyts/thatdataguy-romance-whatsapp-analysis#1]
Cool project! I've tried to get it to work but was wondering if you could share a small sample csv to help me get my own messages in the right format. I've never done anything with Python before so I'm struggling with the syntax.

For example, now I'm running into this error:

AttributeError: 'DataFrameGroupBy' object has no attribute 'num_msg'

Thanks in advance!

**Comment by guyts** [comment]
Yes - for sure, I will try to get to it in the next few days! @jordibruin

**Comment by guyts** [comment]
The error most likely comes from missing columns, as I've added columns to the raw data

**Comment by jordibruin** [comment]
Thanks a lot @guyts!

You mentioned that you fixed the WhatsApp data by cleaning up the new-line data in Excel. I take it this was quite a work of labour with no easy tips and tricks? 😉

**Comment by guyts** [comment]
Yes, @jordibruin unfortunately I didn't try solving this in a faster way since I wanted to meet my own deadline :)
But - I'm sure there could be a way to filter the columns and get values that don't make sense (e.g. don't have a date stamp where it should, due to the new line).
